OpenAPI/kiota/tutorials/dotnet-dependencyinjection.md

```diff
@@ -0,0 +1,288 @@
+---
+title: Use Kiota with dependency injection in .NET
+description: Learn how you can combine Kiota with dependency injection in .NET to build API clients.
+author: svrooij
```

Comment:
add your username to this file please https://github.com/MicrosoftDocs/openapi-docs/blob/main/cspell.json

Reply:
Check 😜
Maybe this should be documented somewhere

Reply:
with #88 we're moving away from that for CI anyway